packages/wallet/src/cip30.ts

```diff
@@ -71,7 +71,13 @@ export const createWalletApi = (
     logger.debug('getting balance');
     try {
       const wallet = await walletReady;
-      const value = await firstValueFrom(wallet.balance.available$);
+      // Review: not including balance.rewardAccounts.{deposit,rewards}, should we?
```

Comment:
Unwithdrawn rewards are conceptually part of the balance, however, the[ CIP indicates](https://cips.cardano.org/cips/cip30/#apigetbalancepromisecborvalue) that it should just be the UTxOs

Reply:
Thanks, should've just taken a look at the cip :man_facepalming: